sdf/concepts/imports.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Imports
+description: :Package imports
+sidebar_position: 400
+---
+
+Dataflows and packages can import function and types from other  packages.  This allows you to reuse code and types across multiple projects.
+
+Importing has two parts:
```

Comment:
Importing -> Importing a package

-> I think you should specify that a package can contain 3 types of information, otherwise it's not obvious what you mean to say.